README.md

```diff
@@ -23,7 +23,7 @@ This is a react based (side) project intended to practice help you in typing fas
 
 ## 📌Live Demo:
 
-![Live Demo](https://github.com/kaiwalyakoparkar/flashtype/blob/master/doc_images/FlashType.avi)
+![Live Demo](https://github.com/Salonijain1248/flashtype/blob/master/doc_images/Flashtype.gif)
```

Comment:
Change this link... If I merge this and you delete the fork then I will lose the gif format.

Reply:
You can change link of Live Demo to (./doc_images/NameIt.gif)